devices/moes.js

```diff
@@ -151,4 +153,45 @@ module.exports = [
         toZigbee: [],
         exposes: [e.battery(), e.illuminance(), e.illuminance_lux().withUnit('lx'), e.humidity(), e.temperature()],
     },
+    {
+        // zigbeeModel: ['b6wax7g0'],
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_b6wax7g0'}],
+        model: 'BRT-100-TRV',
+        vendor: 'Moes',
+        description: 'Thermostatic Radiator Valve',
+        whiteLabel: [{vendor: 'Moes', model: 'BRT-100-TRV'}],
+        ota: ota.zigbeeOTA,
+        onEvent: tuya.onEventSetLocalTime,
+        supports: 'thermostat, temperature',
+        fromZigbee: [fz.ignore_basic_report, fz.ignore_tuya_set_time, fz.moesS_thermostat],
+        toZigbee: [tz.moesS_thermostat_current_heating_setpoint, tz.moesS_thermostat_child_lock,
+            tz.moesS_thermostat_window_detection, tz.moesS_thermostat_temperature_calibration,
+            tz.moesS_thermostat_temperature_calibration, tz.moesS_thermostat_system_mode,
+            tz.moesS_thermostat_boost_heating, tz.moesS_thermostat_boostHeatingCountdownTimeSet,
+            tz.moesS_thermostat_eco_temperature, tz.moesS_thermostat_max_temperature, 
+            tz.moesS_thermostat_min_temperature, tz.moesS_thermostat_schedule, 
+            tz.moesS_thermostat_moesSecoMode, tz.moesS_thermostat_boost_heating_countdown],
+        exposes: [
+            e.battery(), e.child_lock(),
+            exposes.climate()
+                .withLocalTemperature(ea.STATE).withSetpoint('current_heating_setpoint', 5, 35, 0.5, ea.STATE_SET)
+                .withLocalTemperatureCalibration(ea.STATE_SET),
+            e.preset('preset', ea.STATE_SET, ['programming', 'manual', 'temporary_manual', 'holiday'])
```

Comment:
Preset can be added to the climate, after `.withLocalTemperatureCalibration(ea.STATE_SET)` add `.withPreset(['programming', 'manual', 'temporary_manual', 'holiday'])`